components/multiselect/overview.md

```diff
@@ -1,202 +1,142 @@
-                    Placeholder="Enter Balkan country, e.g., Bulgaria"
-                    Width="350px" ClearButton="true" />
-
-@if (Values.Count > 0)
-{
-    <ul>
-        @foreach (var item in Values)
-        {
-            <li>@item</li>
-        }
-    </ul>
-}
-
-@code {
-    List<string> Countries { get; set; } = new List<string>();
-    List<string> Values { get; set; } = new List<string>();
-
-    void SelectHandler()
-    {
-        List<string> PreselectedValues = new List<string>()
-        {
-            "Bulgaria", "Croatia"
-        };
-
-        // create a new reference so that the framework can notify the component to update
-        Values = new List<string>(PreselectedValues);
-    }
-
-    void ClearSelectionHandler()
-    {
-        Values = new List<string>();
-    }
-
-    protected override void OnInitialized()
-    {
-        Countries.Add("Albania");
-        Countries.Add("Bosnia & Herzegovina");
-        Countries.Add("Bulgaria");
-        Countries.Add("Croatia");
-        Countries.Add("Kosovo");
-        Countries.Add("North Macedonia");
-        Countries.Add("Montenegro");
-        Countries.Add("Serbia");
-        Countries.Add("Slovenia");
-
-        // you can also pre-select items here based on data you fetch, not just on a button click
-
-        base.OnInitialized();
-    }
-}
-````
-
-@[template](/_contentTemplates/common/get-model-from-dropdowns.md#get-model-from-dropdowns)
-
-## See Also
-
-  * [Data Binding]({%slug multiselect-databind%})
-  * [Live Demo: MultiSelect](https://demos.telerik.com/blazor-ui/multiselect/overview)
-  * [Live Demo: MultiSelect Validation](https://demos.telerik.com/blazor-ui/multiselect/validation)
-  * [API Reference](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.Components.TelerikMultiSelect-2)
+---
+title: Overview
+page_title: MultiSelect Overview
+description: Overview of the MultiSelect for Blazor.
+slug: multiselect-overview
+tags: telerik,blazor,multiselect,overview
+published: True
+position: 0
+---
+
+# MultiSelect Overview
+
+The <a href="https://www.telerik.com/blazor-ui/multiselect" target="_blank">Blazor MultiSelect component</a> lets the user select several items from the available list. It is similar to a `<select multiple>` in this regard. The MultiSelect offers suggestions as you type and they can be [filtered]({%slug multiselect-filter%}). You can control the list of suggestions through [data binding]({%slug multiselect-databind%}), various appearance settings like [dimensions]({%slug common-features/dimensions%}) and [templates]({%slug multiselect-templates%}).
+
+## Creating MultiSelect
+
+1. Use the `TelerikMultiSelect` tag to add the component to a view, for example, `~/Pages/Index.razor`.
+
+1. Populate the `Data` property with the collection of items that you want to appear in the dropdown.
+
+1. Set the [two-way value binding]({%slug get-started-value-vs-data-binding %}#value-binding) by using `@bind-Value`.

```

Comment:
I think adding this is **great** idea. Let's think about how we can reword it a little. What comes on top of the mind:

Bind the value of the component to a collection that matches in type with the one bound to the `Data` parameter.

I feel like mentioning only the two-way data-binding is not needed, you can also use `Value` + `ValueChanged`